ui/raidboss/data/05-shb/eureka/zadnor.js

```diff
@@ -81,30 +133,1155 @@ export default {
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DAD', capture: false }),
+      condition: (data) => data.ce === 'looks',
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Looks Serpent\'s Edge',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DB1' }),
+      condition: tankBusterOnParty('looks'),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Zadnor Looks Levinbolt',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DB0' }),
+      condition: (data, matches) => data.ce === 'looks' && data.me === matches.target,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Zadnor Looks Thundercall',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5D9C', capture: false }),
+      condition: (data) => data.ce === 'looks',
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Avoid Orbs -> Under Orbs',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Looks Flame',
+      netRegex: NetRegexes.startsUsing({ source: 'Ayida', id: '5DA6', capture: false }),
+      condition: (data) => data.ce === 'looks',
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          // TODO: this is also an aoe, and this is a pretty poor description.
+          en: 'Go to small orb',
+        },
+      },
+    },
     // ***** Taking the Lyon's Share *****
     // ***** The Dalriada *****
+    {
+      id: 'Zadnor Sartauvoir Pyrokinesis',
+      netRegex: NetRegexes.startsUsing({ source: 'Sartauvoir The Inferno', id: '5E7D', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Zadnor Sartauvoir Time Eruption',
+      netRegex: NetRegexes.startsUsing({ source: 'Sartauvoir The Inferno', id: '5E6C', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go to Slow Clocks',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Sartauvoir Reverse Time Eruption',
+      netRegex: NetRegexes.startsUsing({ source: 'Sartauvoir The Inferno', id: '5E6D', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go to Fast Clocks',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Sartauvoir Phenex',
+      netRegex: NetRegexes.startsUsing({ source: 'Sartauvoir The Inferno', id: '5E72', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Bird Dashes',
+        },
+      },
+    },
+    {
+      id: 'Zadnor Sartauvoir Hyperpyroplexy',
+      netRegex: NetRegexes.ability({ source: 'Sartauvoir The Inferno', id: '5E76', capture: false }),
+      infoText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Get Towers',
```

Comment:
Should probably bump this to an Output at this point
```suggestion
          en: 'Get Towers',
          de: 'Türme nehmen',
          fr: 'Prenez les tours',
          ja: '塔を踏む',
          cn: '踩塔',
          ko: '장판 하나씩 들어가기',
```